wolfssl/ssl.h

```diff
@@ -2272,7 +2280,11 @@ WOLFSSL_ABI WOLFSSL_API int wolfSSL_Cleanup(void);
 
 /* which library version do we have */
 WOLFSSL_API const char* wolfSSL_lib_version(void);
+#ifdef WOLFSSL_QT
```

Comment:
Switch on OPENSSL_VERSION_NUMBER instead of WOLFSSL_QT

Reply:
Guard by OPENSSL_VERSION_NUMBER instead of WOLFSSL_QT